chapters/01-introduction.Rmd

```diff
@@ -2,11 +2,19 @@
+In the particle accelerator at CERN for instance, they record a whooping 25 gigabytes of data per second[@cern], resulting from the process of many physical interactions that occur almost simultaneously. It is not feasible to anticipate features of the distribution one observes experimentally, as it is comprised of many different distributions, which result from all the different physical interactions.
+
+To combat this, so called non-parametric methods are used, which do not need a predefined mathematical model. 
+
+The perhaps simplest non-parametric method is the Histogramm. By summing the the data up in discrete bins the underlying parent distribution can be approximated, without needing any knowledge of underlying processes.
+
+However there are also many more sophisticated non-parametric methods, one in particular is Kernel Density Estimation (KDE), which can be looked at as a sort of generalized histogram [@rosenblatt1956remarks].
+
+Histograms tend to produce PDFs that are highly dependent on bin width and bin positioning, meaning the interpretation of the data changes by a lot by two arbitrary parameters. KDE circumvents this problem by replacing each data point with a so called kernel function that specifies how much it influences its neighbouring regions as well. The kernel functions themselves are centered at the data point directly, eliminating the need for arbitrary bin positioning[@duong2001]. Since KDE still depends on kernel bandwidth (instead of bin width), one might argue that this is not a major improvement. However, upon closer inspection, one finds that the underlying PDF does depend less strongly on the kernel bandwidth than histograms on bin width and it is much easier to specify rules for an approximately optimal kernel bandwidth than it is to do so for bin width[@lerner2013]. In addition, by specifying a smooth kernel function, one gets a smooth distribution as well, which is often desirable or even expected from theory.
 
 Due to this increased robustness, KDE is particular useful in High-Energy Physics (HEP) where it has been used for confidence level calculations for the Higgs Searches at the Large Electron Positron Collider (LEP)[@cranmer2000kernel]. However there is still room for improvement and certain more sophisticated approaches to Kernel Density Estimation have been proposed in dependence on specific areas of application[@cranmer2000kernel].
```

Comment:
[4] yes true, however it is mostly used in fits for example. I would mention this as well